services/src/datasets/external/nfdi/mod.rs

```diff
@@ -366,7 +366,8 @@ impl NFDIDataProvider {
     fn raster_loading_template(info: &RasterInfo, rd: &RasterResultDescriptor) -> GdalLoadingInfo {
         let part = GdalLoadingInfoPart {
             time: info.time_interval,
-            params: GdalDatasetParameters {
+            // TODO: set to None when there is no data
```

Comment:
does this have to happen here, or can the caller of the function be responsible for calling it only when there is data. otherwise this template isn't needed anyway

Reply:
i do not know. i guess the method  is only called for valid data.

Reply:
then I'd say you can remove the todo

Reply:
49d1a42